mava/systems/system.py

```diff
@@ -58,22 +57,17 @@ def _make_config(self) -> None:
         """
         for component in self._design.get().values():
             config_class = component.__init__.__annotations__["config"]
-            input = {component.name(): config_class()}
-            self.config.add(**input)
 
-    @abc.abstractmethod
-    def design(self) -> Tuple[DesignSpec, Dict]:
-        """System design specifying the list of components to use.
-
-        Returns:
-            Tuple[system callback components, system config].
-        """
+            if config_class is not SimpleNamespace:
```

Comment:
Why did we add an if statement here? We still need to config right?

Reply:
I just passed the systempy from develop branch

Reply:
I will check how to fix it